src/app/asset-page/artstor-viewer/artstor-viewer.component.ts

```diff
@@ -407,16 +407,19 @@ export class ArtstorViewerComponent implements OnInit, OnDestroy {
         });
 
         this.osdViewer.addOnceHandler("tile-drawn", () => {
-          // Get all thumbnails in the reference strip
-          let referenceStripThumbnails = document.querySelectorAll(".referencestrip .openseadragon-canvas");
-
-          referenceStripThumbnails.forEach((referenceStripThumbnail, index) => {
-            referenceStripThumbnail.addEventListener("keydown", (event: KeyboardEvent) => {
-             if (event.keyCode === 13) {
-               this.osdViewer.goToPage(index)
-             }
-           })
-          })
+            let mainContentCanvas: HTMLElement = <HTMLElement>document.getElementsByTagName("canvas")[0];
+            mainContentCanvas && mainContentCanvas.setAttribute("aria-label", this.assets[0].title ? this.assets[0].title : this.asset[0].description);
```

Comment:
I thought the asset array could be empty when presenting from an image group page. But I could possibly be misremembering. Do you know if this is all good when presenting the group?

Reply:
Tested, assets still good when entering from an image group.

Reply:
🎉